src/GistEntropyUtils.h

```diff
@@ -0,0 +1,88 @@
+#ifndef GIST_ENTROPY_UTILS_H
+#define GIST_ENTROPY_UTILS_H
+
+#include <cstdlib>
+#include <vector>
+#include "Vec3.h"
+#include <cmath>
+
+typedef std::vector<float> Farray;
+
+#define GIST_HUGE 10000.0
```

Comment:
It would be better to make this a `const double` in a `GistEntropyUtils` namespace, see above.